frameworks/digital-outcomes-and-specialists/questions/services/businessAnalystDayRate.yml

```diff
@@ -7,3 +7,7 @@ type: pricing
 fields:
   minimum_price: businessAnalystPriceMin
   maximum_price: businessAnalystPriceMax
+
+validations:
+  - name: answer_required
```

Comment:
These price ones won't work but that's fixed in https://github.com/alphagov/digitalmarketplace-frameworks/pull/130